spec/system/page_objects/components/select_kit.rb

```diff
@@ -75,29 +74,29 @@ def has_no_option_value?(value)
       end
 
       def expand
-        collapsed_component.find(":not(.is-expanded) .select-kit-header", visible: :all).click
```

Comment:
`collapsed_component` already looks for `:not(.is-expanded)` so this is not necessary.